controllers/schools.js

```diff
@@ -17,9 +18,15 @@ router.get('/', async (req, res, next) => {
 		params.qs._id = { $ne: res.locals.currentSchool };
 	}
 	try {
-		const schools = await api(req).get('/schools/', params);
+		let schools = [];
+		if (req.query.$limit === 'false') {
```

Comment:
this is needed because in static script teamMembers we have a call to get all data directly to the /schools endpoint.